UserDetailsClient/UserDetailsClient.Core/App.xaml.cs

```diff
@@ -38,19 +42,24 @@ public App()
             MainPage = new NavigationPage(new MainPage());
         }
 
-        protected override void OnStart()
-        {
-            // Handle when your app starts
-        }
+        //public static Page GetMainPage()
+        //{
+        //    var rootPage = new MasterPage();
+        //    return rootPage;
+        //}
```

Comment:
remove?

Reply:
removed it

Reply:
thanks!

---
In reply to: [280844200](https://github.com/Azure-Samples/active-directory-b2c-xamarin-native/pull/85#discussion_r280844200) [](ancestors = 280844200)